packages/file_selector/file_selector_platform_interface/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
+
+* Changes XTypeGroup initialization from final to const.
```

Comment:
This test-only change doesn't need a CHANGELOG entry, so this file can be reverted.